hetmech/test_matrix.py

```diff
@@ -0,0 +1,28 @@
+import hetio.readwrite
+from .matrix import metaedge_to_adjacency_matrix
+
+'''
+Test the functionality of metaedge_to_adjacency_matrix in
+generating sparse matrices vs numpy arrays. Uses same test
+data as in test_degree_weight.py Figure 2D of Himmelstein &
+Baranzini (2015) PLOS Comp Bio.
+https://doi.org/10.1371/journal.pcbi.1004259.g002
+'''
+
+url = 'https://github.com/dhimmel/hetio/raw/{}/{}'.format(
+    '9dc747b8fc4e23ef3437829ffde4d047f2e1bdde',
+    'test/data/disease-gene-example-graph.json',
+)
+graph = hetio.readwrite.read_graph(url)
+
+
+def test_sparse_matrix():
+    mat = metaedge_to_adjacency_matrix(graph, 'GiG')
+    assert all([mat[2][0, 4], mat[2][0, 2], mat[2][6, 2], mat[2][2, 0]])
```

Comment:
This is on the right track: it checks for the boolean value of certain cells. Let's be even more explicit by testing that the matrix object is exactly equal. You'll want to construct an actual test matrix like [here](https://github.com/greenelab/hetmech/blob/df8a1fca2193f375e8c8c79be76ac8ab255699ea/hetmech/test_diffusion.py#L14-L19).